planemo/options.py

```diff
@@ -727,6 +727,16 @@ def split_job_and_test():
     return click.option("--split_test/--no_split_test", default=False, help="Write workflow job and test definitions to separate files.")
 
 
+def from_invocation():
+    return planemo_option(
+        "--from_invocation/--not_from_invocation",
```

Comment:
I think ``--from_path`` or ``--from_uri`` instead of ``--not_from_invocation`` feels more natural 😄 .